cloud-azure/src/main/resources/templates/arm-dbstack.ftl

```diff
@@ -0,0 +1,179 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dbServerName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the database server resource."
+            }
+        },
+        "skuTier": {
```

Comment:
Add description metadata?

Reply:
Yes, a bunch of descriptions were missing, thanks for pointing out that.